frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/Add_HPA_Action.feature

```diff
@@ -0,0 +1,75 @@
+Feature: Add HPA action and topology sidebar modifications
+	As a user, I want to add a HPA to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment
+    And user has added CPU resource limit
+    And user has added Memory resource limit
+
+@regression
+Scenario: Add HPA form
+   Given user is in topology
+   When user opens sidebar of workload
+   And user opens action menu
+   And user clicks on Add Horizontal Pod Autoscaler option 
+   Then user can see Add Horizontal Pod Autoscaler page opens with form view selected
+   And user can see name field 
+   And user can see minimum and maximum pods with increasing and decreasing button
+   And user can see CPU Utilization
+   And user can see Memory Utilization
+   And user can see save and cancel button at the end
+
+@regression
+Scenario: Add HPA to deployment from action menu
+   Given user is in topology
+   When user opens sidebar of workload
+   And user opens action menu
+   And user clicks on Add Horizontal Pod Autoscaler option 
+   And user sees Add Horizontal Pod Autoscaler page opens with form view selected
+   And user adds name 'example' 
+   And user assigns minimum pods value to 2 
+   And user assigns maximum pods value to 5
+   And user gives CPU Utilizationvalue as 60
+   And user gives Memory Utilization value as 30
+   And user clicks on save to get back to topology page
+   And user opens the sidebar
+   Then user can see two pods under pod section in details tab
```

Comment:
I would suggest to add a when statement where user will click on Details tab on the sidebar. Just to be safe.